### ISSUE TITLE
NodeJS command line app result window closes immediately

### ISSUE THREAD

**Issue by rajakondla** [Microsoft/nodejstools#1998]
##### Actual Behavior 

- NTVS Version:
- Visual Studio Version: 
- Node.js Version: 

##### Steps to Reproduce
I am using vs 2017 community edition.
I am new to nodeJS with TypeScript. I have created nodeJS TypeScript command line application. The result window is closing immediately after running the application. I have tried checking the check box (Wait for input when process exits normally under Tools->options->nodeJS tools) but still window closes immediately. Please suggest if I am missing anything.

**Comment by paulvanbrenk** [comment]
Yes, we're investigating how to re-enable that checkbox for node 8 (and newer). YOu should be able to set a breakpoint and have the program stop that way.

**Comment by paulvanbrenk** [comment]
Yes, we're investigating how to re-enable that checkbox for node 8 (and newer). YOu should be able to set a breakpoint and have the program stop that way.

**Comment by amcasey** [comment]
@jessetrinity If this still repros, we probably just need to adjust the arguments to our PressAnyKey exe.

**Comment by jessetrinity** [comment]
It doesn't look like the PressAnyKey process is even being started when debugging. It does launch and respect the options when 'Start Without Debugging (Ctrl + F5)' is selected. I will keep looking to see what can be done in the former case.

**Comment by paulvanbrenk** [comment]
Last time I looked at this, this was something that needs to be handled in the debugger code.
________________________________

**Comment by Janvier123** [comment]
Ive had "C:\Program Files\nodejs\node.exe" checked as "Run this program as an administrator" for an issue. Unchecked the box => fixed

**Comment by alessiot89** [comment]
this thing is still bugged.
